tgui/packages/tgui/interfaces/TraitorBackstoryMenu.jsx

```diff
@@ -0,0 +1,718 @@
+import { useBackend, useLocalState } from '../backend';
```

Comment:
would be nice if this was ported to typescript, but not a requirement and i'll still be fine with merging this PR without that

Reply:
Yeah unfortunately Im too dumb to do that at the moment but one day I will not be.